src/components/PDFThumbnail/index.native.tsx

```diff
@@ -22,16 +28,27 @@ function PDFThumbnail({previewSourceURL, style, isAuthTokenRequired = false, ena
                         singlePage
                         style={sizeStyles}
                         onError={(error) => {
-                            if (!('message' in error && typeof error.message === 'string' && error.message.match(/password/i))) {
-                                return;
+                            if (onLoadError) {
+                                onLoadError();
                             }
-                            if (!onPassword) {
+                            if ('message' in error && typeof error.message === 'string' && error.message.match(/password/i) && onPassword) {
+                                onPassword();
                                 return;
                             }
-                            onPassword();
+                            setHasError(true);
                         }}
                     />
                 )}
+                {hasError && (
+                    <View style={[styles.justifyContentCenter, styles.pdfErrorPlaceholder, styles.alignItemsCenter]}>
+                        <Icon
+                            src={Expensicons.ReceiptSlash}
+                            width={variables.receiptPlaceholderIconWidth}
+                            height={variables.receiptPlaceholderIconHeight}
+                            fill={theme.icon}
+                        />
+                    </View>
+                )}
```

Comment:
The lines of code below are common in both files; we can create a common file for this.

```js
<View style={[styles.justifyContentCenter, styles.pdfErrorPlaceholder, styles.alignItemsCenter]}>
    <Icon
        src={Expensicons.ReceiptSlash}
        width={variables.receiptPlaceholderIconWidth}
        height={variables.receiptPlaceholderIconHeight}
        fill={theme.icon}
    />
</View>
```